locations/spiders/starbucks.py

```diff
@@ -16,7 +16,20 @@ def start_requests(self):
         with open('./locations/searchable_points/us_centroids_50mile_radius.csv') as points:
             next(points)
             for point in points:
-                row = point.split(',')
+                row = point.strip().split(',')
```

Comment:
Instead of splitting on comma, can you use `csv.reader` or `csv.DictReader`? That'd make it a bit cleaner and more robust.